src/lib/sql/sql_identifier_resolver.hpp

```diff
@@ -14,7 +14,7 @@ class AbstractExpression;
 
 struct SQLIdentifierContextEntry final {
   std::shared_ptr<AbstractExpression> expression;
-  std::optional<SQLIdentifier> identifier;
+  std::vector<SQLIdentifier> identifiers;
```

Comment:
Adding on @mrzzzrm comment:

What is the result of `SELECT a AS x FROM t GROUP BY t.x`?

Reply:
This crashes, as well as SQLite. I added a test for this.